docs/application/native/api/iot-headed/5.0/group__Edje__Part__Drag.html

```diff
@@ -294,7 +294,7 @@
   </table>
   </dd>
 </dl>
-<dl class="return"><dt><b>Returns:</b></dt><dd><code>true</code> on success, <code>false</code> otherwise </dd></dl>
+<dl class="return"><dt><b>Returns:</b></dt><dd><code>true</code> on success, <code>false</code> otherwise  2.3 </dd></dl>
```

Comment:
![image](https://user-images.githubusercontent.com/9958374/102871846-48a2d400-4482-11eb-8edc-343e42eec5fd.png)

I'm confused with this sentence.

Is this update really intended?

Many places are updated in this way, but I just point out this only.

Reply:
I think.. it is bug generated by EFL versioning script.. Will check it with EFL team...

./group__Edje__Object__Scale.html:<dl class="return"><dt><b>Returns:</b></dt><dd><code>true</code> on success, <code>false</code> otherwise  2.3 </dd></dl>
./group__Edje__Part__Drag.html:<dl class="return"><dt><b>Returns:</b></dt><dd><code>true</code> on success, <code>false</code> otherwise  2.3 </dd></dl>
./group__Edje__Part__Drag.html:<dl class="return"><dt><b>Returns:</b></dt><dd><code>true</code> on success, <code>false</code> otherwise  2.3 </dd></dl>
./group__Edje__Part__Drag.html:<dl class="return"><dt><b>Returns:</b></dt><dd><code>true</code> on success, <code>false</code> otherwise  2.3 </dd></dl>
./group__Edje__Part__Drag.html:<dl class="return"><dt><b>Returns:</b></dt><dd><code>true</code> on success, <code>false</code> otherwise  2.3 </dd></dl>
./group__Edje__Part__Drag.html:<dl class="return"><dt><b>Returns:</b></dt><dd><code>true</code> on success, <code>false</code> otherwise  2.3 </dd></dl>
./group__Edje__Part__Drag.html:<dl class="return"><dt><b>Returns:</b></dt><dd><code>true</code> on success, <code>false</code> otherwise  2.3 </dd></dl>
./group__Edje__Part__Drag.html:<dl class="return"><dt><b>Returns:</b></dt><dd><code>true</code> on success, <code>false</code> otherwise  2.3 </dd></dl>
./group__Edje__Part__Drag.html:<dl class="return"><dt><b>Returns:</b></dt><dd><code>true</code> on success, <code>false</code> otherwise  2.3 </dd></dl>
./group__Edje__Part__Drag.html:<dl class="return"><dt><b>Returns:</b></dt><dd><code>true</code> on success, <code>false</code> otherwise  2.3 </dd></dl>
./group__Edje__Part__Drag.html:<dl class="return"><dt><b>Returns:</b></dt><dd><code>true</code> on success, <code>false</code> otherwise  2.3 </dd></dl>
./group__Edje__Object__Part.html:<dl class="return"><dt><b>Returns:</b></dt><dd><code>true</code> on success, <code>false</code> otherwise  2.3 </dd></dl>
./group__Edje__Object__Group.html:<dl class="return"><dt><b>Returns:</b></dt><dd><code>true</code> on success, <code>false</code> otherwise  2.3 </dd></dl>
./group__Edje__Object__Group.html:<dl class="return"><dt><b>Returns:</b></dt><dd><code>true</code> on success, <code>false</code> otherwise  2.3 </dd></dl>

Reply:
https://review.tizen.org/gerrit/#/c/platform/upstream/efl/+/250479/